src/main/java/hudson/plugins/analysis/graph/CategoryBuildResultGraph.java

```diff
@@ -234,7 +237,18 @@ protected JFreeChart createChart(final GraphConfiguration configuration, final R
         }
         return valuesPerBuild;
     }
-
+    private boolean passesFilteringByParameter(final AbstractBuild<?, ?> build, final String parameterName, final String parameterValue){
+        if (parameterName == null){
```

Comment:
Can you please check your formatter preferences? Whitespace between ( and {. Maybe it would be safe to select your chagnes and appy auto-reformat).